### PR TITLE
docs: reword the existing mirrors section

### PR DESCRIPTION
**Pull request by galargh**
The blog post you created is amazing ❤️ I think it should be the first thing people see.

I let myself reword the `existing mirrors` section a bit. I tried to make it more explicit about what is what to make it more entry-level user friendly. Let me know what you think.

Could we also merge https://github.com/ipfs/distributed-wikipedia-mirror/pull/97? I think that refactor complements the blogpost and the README perfectly as it makes `snapshot-hashes.yml` use the same words as the other documents.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,10 +9,10 @@ Putting Wikipedia Snapshots on IPFS and working towards making it fully read-wri
 <br />
 
 ## Existing Mirrors
-  
-Below are [DNSLink names](https://docs.ipfs.io/concepts/dnslink/) that point at the latest CIDs from [snapshot-hashes.yml](https://github.com/ipfs/distributed-wikipedia-mirror/blob/main/snapshot-hashes.yml).
 
-There  various way one can access IPFS mirror (either with DNSLink, or direct CID), [read about them here](https://blog.ipfs.io/2021-05-31-distributed-wikipedia-mirror-update/#improved-access-to-wikipedia-mirrors).
+There are various ways one can access the mirrors: through a DNSLink, public gateway or directly with a CID. You can [read all about the available methods here](https://blog.ipfs.io/2021-05-31-distributed-wikipedia-mirror-update/#improved-access-to-wikipedia-mirrors).
```
Comment (lidel) [inline README.md:13]:
```suggestion
There are various ways one can access the mirrors: through a [DNSLink](https://docs.ipfs.io/concepts/glossary/#dnslink), public [gateway](https://docs.ipfs.io/concepts/glossary/#gateway) or directly with a [CID](https://docs.ipfs.io/concepts/glossary/#cid). 

You can [read all about the available methods here](https://blog.ipfs.io/2021-05-31-distributed-wikipedia-mirror-update/#improved-access-to-wikipedia-mirrors).
```

File: README.md [hunk 2]
```diff
@@ -9,10 +9,10 @@ Putting Wikipedia Snapshots on IPFS and working towards making it fully read-wri
...
+
+### [DNSLink](https://docs.ipfs.io/concepts/dnslink/)s
```
Comment (lidel) [inline README.md:15]:
```suggestion
### DNSLinks
```

File: README.md [hunk 3]
```diff
@@ -23,10 +23,13 @@ There  various way one can access IPFS mirror (either with DNSLink, or direct CI
 - https://ru.wikipedia-on-ipfs.org
 - https://fa.wikipedia-on-ipfs.org
 
+### [CID](https://docs.ipfs.io/concepts/content-addressing/)s
```
Comment (lidel) [inline README.md:26]:
```suggestion
### CIDs
```

**Review by lidel — APPROVED** [review]
Thanks! 

- Merged https://github.com/ipfs/distributed-wikipedia-mirror/pull/97 – thanks for rebasing it! 
- Made small tweaks (it is safer to link to glossary than anything else) but lgtm, merging this one as well.

**Comment by galargh** [comment]
Thank you 😊

**Pull request merged**